contracts/HybridCustody.cdc

```diff
@@ -487,6 +501,22 @@ pub contract HybridCustody {
             return nil
         }
 
+        access(contract) fun parentRemoveChildCallback(parent: Address) {
+            if !self.childCap.check() {
+                return
+            }
+
+            let child = self.childCap.borrow()!
```

Comment:
NP: would prefer to see a Type declaration since where calling `check()` next where I immediately think `child` is a `Capability` but realized it's `&ChildAccount` since we call `borrow()` on the cap here. More about this further down